osm2pgsql.lua

```diff
@@ -786,6 +789,15 @@ function osm2pgsql.process_relation(object)
         return
     end
 
+-- Mark some disputes as unrecognized to hide them by default
```

Comment:
Nit: add incline comments to say plainly what disputed each of these `B##` are for